Assets/InventoryVisualsManager.cs

```diff
@@ -2,6 +2,7 @@ namespace CT6RIGPR
 {
     using DG.Tweening;
     using UnityEngine;
+    using static CT6RIGPR.Constants;

```

Comment:
Unless I'm mistaken and I've missed this, I don't see any constants being used here. Do we need this using?-

Reply:
Oh there is the powerup type enum, is that being used here?

Reply:
An earlier version did but not this one so removed.